Automattic/jetpack
# Jetpack Plugin: Add VideoPress upgrade page

#### Changes proposed in this Pull Request:

* This PR adds the description for VideoPress to `Jetpack::get_products_for_purchase` (from `https://github.com/Automattic/jetpack/blob/master/projects/plugins/jetpack/class.jetpack.php`), and also a new upgrade page that will display the copy added to the method above.
~~_Note: Every upgrade page has its own illustration. In the case of this page, the illustration is not yet ready. So we'll be using the same illustration as Search, then we'll replace it as soon as the new illustration is ready. This does not affect the testing/codereview._~~

#### Jetpack product discussion


Context: pbNhbs-18H-p2

#### Does this pull request change what data or activity we track or use?


Not directly.

#### Testing instructions:






| Mobile | Phablet | Desktop |
|--------|-------- |---------| 
| ![image](https://user-images.githubusercontent.com/5714212/138455735-65e51b7f-8895-487d-9b10-27c3dd8abdd8.png) | ![image](https://user-images.githubusercontent.com/5714212/138455662-866f746e-2d43-451a-9e7c-345bdb4e6fe5.png) | ![image](https://user-images.githubusercontent.com/5714212/138455812-133857df-ac27-4f2f-9811-7fef92185d9f.png) |

*Testing the page*
* Go directly to `/wp-admin/admin.php?page=jetpack#/product/videopress` and make sure this page exists and shows the copy added to `Jetpack::get_products_for_purchase`, and matches the screenshot above

*Testing the upgrade button*
* Go to the Jetpack Dashboard
* Make sure you don't have VideoPress or any other product that includes VideoPress (right now it's only Complete)
* Scroll down to the Performance section and click "Activate" under the VideoPress card. After activating, an upgrade button is displayed
* Click on that button and make sure it directs you to `/wp-admin/admin.php?page=jetpack#/product/videopress`, and that page shows the right copy, illustration, and it matches the screenshot above.
